.github/ISSUE_TEMPLATE/Distribution_Request.md

```diff
@@ -14,7 +14,7 @@ assignees: ''
 - Processor Architecture (One per request): 
 - [ ] The version of the Distribution is supported for at least one year.
 - [ ] The version of the Distribution is not an [interim release](https://ubuntu.com/about/release-cycle) or equivalent.
-- [ ] The version of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
+- [ ] The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
```

Comment:
Why do we reference to Linux?

Reply:
This is a distribution request... which implies Linux.  macOS... is macOS... Windows, we will automatically pickup new versions...